backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -593,25 +593,32 @@ def make_path_object(self, data: typing.Dict[str, typing.Any]) -> object:
         return CommentPath(**data)
 
 
-class KnownMemberQuerySchema(marshmallow.Schema):
+class KnownMembersQuerySchema(marshmallow.Schema):
     acp = StrippedString(
         example="test", description="search text to query", validate=acp_validator, required=True
     )
 
     exclude_user_ids = StrippedString(
         validate=regex_string_as_list_of_int,
         example="1,5",
-        description="comma separated list of excluded user",
+        description="comma separated list of excluded users",
     )
+
     exclude_workspace_ids = StrippedString(
         validate=regex_string_as_list_of_int,
         example="3,4",
-        description="comma separated list of excluded workspace: user of this workspace are excluded from result",
+        description="comma separated list of excluded workspaces: members of this workspace are excluded from the result",
```

Comment:
Maybe add note somewhere about the fact that exclude_workspace_ids and include_workspace_ids are mutually exclusive. Theses description are use in swagger autogenerated documentation.

Reply:
ah, right

Reply:
done